source/index.ts

```diff
@@ -0,0 +1,9 @@
+import express from "express";
+import { config } from "dotenv";
+
+config();
+
+const port = process.env.PORT || 3333;
```

Comment:
Another convention that is good to say is: the server's port should be higher because on this low range (1000 - 5000) is common to have conflicts with OS ports.
I would suggest to change the line 6 to:
`cost port = process.env.PORT || 8000`